platform/linux-dpdk/include/odp/api/plat/version_types.h

```diff
@@ -0,0 +1 @@
+../../../../../linux-generic/include/odp/api/plat/version_types.h
```

Comment:
We were able to get rid of the symlinks for /platform/linux-dpdk/odp_xxx.c because we just changed Makefile.am to build /platform/linux-generic/odp_xxx.c, but we still need these symlinks here, right?

The only way to avoid this duplication of shared/common code is to move it outside of the /platfom/xxx/ directory so that multiple platforms can share.  This is a long-term goal -- just want to think about possible solutions...